templates/org/home.tmpl

```diff
@@ -40,8 +40,11 @@
 			<div class="ui {{if .ShowMemberAndTeamTab}}eleven wide{{end}} column">
 				{{if .ProfileReadme}}
 					<div id="readme_profile" class="markup">{{.ProfileReadme | Str2html}}</div>
+					{{template "explore/repo_search" dict "ctxData" . "AutoFocus" false}}
+				{{else}}
+					{{template "explore/repo_search" dict "ctxData" . "AutoFocus" true}}
```

Comment:
Is ProfileReadme a boolean? If yes:

```
{{template "explore/repo_search" dict "ctxData" . "AutoFocus" (not .ProfileReadme}}
```

Could also rename to `IsProfileReadme` to make it more clear that it's a boolean.

Reply:
Actually no, I see it's string so it would need a boolean convertion.

Reply:
> Actually no, I see it's string so it would need a boolean convertion.

No. It is always right to use `not .StringValue` in Golang Template: zero-value means false

Reply:
Ok then let's do it.

Reply:
https://github.com/go-gitea/gitea/pull/27982/files#r1390949515